src/engine/N3Base/N3UIImage.h

```diff
@@ -29,7 +29,7 @@ class CN3UIImage : public CN3UIBase {
 
     float         m_fAnimFrame;     // 1�ʴ� �׷���� �̹����� ��
     int           m_iAnimCount;     // Animate�Ǵ� �̹��� ��(Load�� �� child�� ���ڷ� �ľ�)
-    int           m_fCurAnimFrame;  // ���� �׷��� animate frame index (�Ҽ�� ��� �������)
+    float         m_fCurAnimFrame;  // ���� �׷��� animate frame index (�Ҽ�� ��� �������)
```

Comment:
That's need a funky one. I believe they were testing with it officially, the fact that the variable was starting with `f` to indicate a `float`, but was declared as `int`. Good catch and important fix it is!

Reply:
Thank you! It was an annoying issue to debug indeed.